eclipse-collections/src/main/java/org/eclipse/collections/impl/list/mutable/MultiReaderFastList.java

```diff
@@ -917,6 +918,40 @@ public int lastIndexOf(Object o)
                         + "If you would like to use a ListIterator, you must either use withReadLockAndDelegate() or withWriteLockAndDelegate().");
     }
 
+    /**
+     * @since 10.0
+     */
+    @Override
+    public void replaceAll(UnaryOperator<T> operator)
+    {
+        this.acquireWriteLock();
+        try
+        {
+            this.delegate.replaceAll(operator);
+        }
+        finally
+        {
+            this.unlockWriteLock();
+        }
+    }
+
+    /**
+     * @since 10.0
+     */
+    @Override
+    public void sort(Comparator<? super T> comparator)
+    {
+        this.acquireWriteLock();
+        try
+        {
+            this.delegate.sort(comparator);
+        }
+        finally
+        {
+            this.unlockWriteLock();
+        }
+    }
```

Comment:
Is sortThis() overridden? I was expecting sort() to be the method before sortThis().